### ISSUE TITLE
Unable to drag files into Cinny Flatpak (sandbox permission issue)

### ISSUE THREAD

**Issue by WfKe9vLwSvv7rN** [flathub/in.cinny.Cinny#3]
TL;DR: By granting appropriate filesystem permissions in Flatseal or with `flatpak override`, this issue is resolved. Whether that permission (`filesystem=home`) should be granted by default is debatable (convenience vs security), but Cinny should certainly better handle the case where the necessary filesystem access is denied.

## Current behavior when filesystem access permission is not granted

_Using Cinny Flatpak v2.0.4._

Dragging a file into Cinny (from a file manager) _succeeds_ in attaching the file but shows a size of "n/a". This also happens with copying an image _file_ and pasting it into Cinny.

![image](https://user-images.githubusercontent.com/96372288/173171892-49e313f0-8b2a-458c-b6e9-945c123f6b8b.png)

![image](https://user-images.githubusercontent.com/96372288/173171915-f34b10f0-7c46-487f-880a-946d5bb553b1.png)

Generic file attachments cause behavior identical to [folder attachments](https://github.com/ajbura/cinny/issues/621).

Attempting to send a broken image attachment will lock out the text field (`disabled` attribute and `cursor: not-allowed;` style). This state is recoverable by removing the attachment (which remains possible after a send attempt unlike with folders and generic files) and then successfully sending a message, such as a text message or a valid attachment obtained through the file chooser button. If text was not entered prior to attempting to send the broken image, it is possible to insert emoji via the emoji picker even in the disabled state of the text field (this is probably a separate bug).

![image](https://user-images.githubusercontent.com/96372288/173172352-c2cd3af5-7250-4c85-b5be-75f41f5dcb27.png)

I have not tested the behavior of any other special file types (audio, video).

### Expected behavior

Broken file attachments should be rejected immediately and never appear to be sendable. Something along the lines of checking for 0-byte files should solve this problem.
